examples/backend/src/config/database.data-source.ts

```diff
@@ -0,0 +1,34 @@
+import * as secrets from 'docker-secrets-nodejs';
+
+import { DataSource } from 'typeorm';
+import { Environment, EnvironmentType } from './environment';
+
+const isDev = Environment.current() === EnvironmentType.Development;
+const isLocal = Environment.current() === EnvironmentType.Local;
+const isTS = !!process.env.IS_TS;
+
+console.log('**** Environment: ', Environment.description(), ' ****');
+
+const DOCKER_DB_NAME = 'mariadb';
+
+const dataSource = new DataSource({
+    name: 'default',
+    type: 'mysql',
+    host: isDev ? 'localhost' : DOCKER_DB_NAME,
+    port: 3306,
+    username: 'root',
+    password: isDev ? 'test1234' : secrets.get('db-root-password'),
```

Comment:
Same.

Reply:
I've created an issue for this one, as @joselufo also didn't like this approach: https://github.com/mobilejazz/harmony-typescript/issues/105